src/main/resources/com/mathworks/ci/tools/MatlabInstaller/help-products.html

```diff
@@ -0,0 +1,15 @@
+<div>
```

Comment:
For these two help items, we need to follow the same pattern/tone we have already used for other fields.